lms/static/scripts/frontend_apps/config.ts

```diff
@@ -53,6 +53,7 @@ export type InstructorConfig = {
   editingEnabled: boolean;
   gradingEnabled: boolean;
   students: StudentInfo[] | null;
+  scoreMaximum?: number;
```

Comment:
I think we should handle the case of "we don't know the score Max" explicitly instead of offereing a default (10?) from the backend.

Not sure if `?` is the right type hint, or is it `number | null`?

Reply:
Adding `?` to a property is a shorthand for adding `| undefined` to the type. If the backend omits `scoreMaximum` if unknown, then `scoreMaximum?: number` is correct. If it always sets the field, but might set it to `None`, then `scoreMaximum: number | null` is correct.

Reply:
Gone for the former, the key will always be there but it might be null